### ISSUE TITLE
Edge saved personal info autofill not working for QSelect

### ISSUE THREAD

**Issue by gabriellefuchs** [quasarframework/quasar#17383]
### What happened?

When Edge's 'Save and fill basic info' setting is enabled and saved personal info is used to auto fill a QSelect in a form the below error occurs:
`U2.value(...).toLocaleLowerCase is not a function`

### What did you expect to happen?

No thrown error and select to auto fill

### Reproduction URL

https://codepen.io/gabriellefuchs/pen/XWLXoJG?editors=0010

### How to reproduce?

1. In Edge, enable the Personal info setting 'Save and fill basic info'
2. Go to the provided reproduction link. 
3. Fill out the form and select save. 
4. Click Reset
5. Select the last used option in the Saved personal info drop down 

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Components (quasar)

### Platforms/Browsers

Microsoft Edge

### Quasar info output

```Shell
quasar 2.16.4
vite 5.3.1
vue 3.4.28

Operating System        Linux(6.9.8-100.fc39.x86_64) - linux/x64
NodeJs                  20.12.2

Global packages    
  NPM                   10.5.0
  yarn                  Not installed
  pnpm                  9.3.0
  bun                   Not installed
  @quasar/cli           2.4.1
```

### Relevant log output

```Shell
TypeError: gt.value(...).toLocaleLowerCase is not a function
```

### Additional context

Found #16698 which has a commit that would resolve the issue but has been inactive since Dec.

https://github.com/user-attachments/assets/c1bfcab4-b685-43b6-9f3d-e95ca2879b28

**Comment by kalvenschraut** [comment]
@rstoenescu  any  idea how we can get this resolved or what is blocking the issue? Fix seems simple and we are having to maintain a patch with this fix in place for several months now.

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.17.2

**Comment by rstoenescu** [comment]
@kalvenschraut Sorry that it took so much and for bringing this into my attention again!
